style.css

```diff
@@ -80,3 +97,191 @@ main {
 .connect-icons a:hover {
   background-color: #babdda;
 }
+
+/*
+  Works Section
+*/
+
+.works {
+  display: grid;
+  padding: 8%;
+  padding-top: 35%;
+  background-color: #e5e5e5;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 12px;
+  background-color: #fff;
+  margin: 15% 0;
+  padding: 5%;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.card-cover {
+  width: 100%;
+}
+
+.card-title {
+  color: #172b4d;
+  font-size: 32px;
+  font-weight: 700;
+}
+
+.card-details {
+  display: flex;
+  align-items: center;
+  gap: 12px;
+  font-size: 13px;
+  font-weight: 600;
+  height: 32px;
+}
+
+.prim-detail-cap {
+  color: #344563;
+}
+
+.sec-detail-cap {
+  color: #7a869a;
+}
+
+.card .description {
+  font-size: 15px;
+  line-height: 24px;
+  font-weight: 400;
+  color: #344563;
+}
+
+.techstack {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+  margin: 3% 0;
+}
+
+.techstack li {
+  padding: 4px 12px;
+  background-color: #ebebff;
+  border-radius: 8px;
+  font-size: 12px;
+  color: #6070ff;
+}
+
+.techstack li:hover {
+  box-shadow:
+    1px 1px 5px rgba(0, 0, 0, 0.1),
+    -1px -1px 5px rgba(0, 0, 0, 0.1);
+}
+
+.works .card a {
+  color: #6070ff;
+  border: 1px solid #6070ff;
+  padding: 12px;
+  border-radius: 8px;
+  font-size: 17px;
+}
+
+.works .card a:hover {
+  background-color: #6070ff;
+  color: #fff;
+  box-shadow:
+    1px 1px 5px rgba(0, 0, 0, 0.3),
+    -1px -1px 5px rgba(0, 0, 0, 0.3);
+}
+
+/* About Myself Section */
+
+.about-myself {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  align-items: flex-start;
+  padding: 8%;
+  background-color: #fff;
+  padding-top: 30%;
+}
+
+.about-title {
+  color: #172b4d;
+  font-size: 40px;
+}
+
+.about-description {
+  color: #344563;
+}
+
+.resume-link {
+  color: #6070ff;
+  border: 1px solid #6070ff;
+  padding: 12px;
+  border-radius: 8px;
+  font-size: 17px;
+}
+
+.resume-link:hover {
+  background-color: #6070ff;
+  color: #fff;
+  box-shadow:
+    1px 1px 5px rgba(0, 0, 0, 0.3),
+    -1px -1px 5px rgba(0, 0, 0, 0.3);
+}
```

Comment:
- [ ] Dear @dev89s, It's great you have a hover effect and it looks great. But please notice we are expected to add a `Active` and `Pressed` effect also. 

![Screenshot 2023-02-17 220425](https://user-images.githubusercontent.com/69077061/219764051-b10f1c13-1154-4cae-9b70-3fd436885729.png)